src/apigateway/schema.js

```diff
@@ -4,35 +4,39 @@ const yaml = require('js-yaml');
 const RefParser = require('json-schema-ref-parser');
 const mergeAll = require('json-schema-merge-allof');
 
-const defaultEnconding = 'utf8';
-
 class Schema {
     static fromFilePath(schemaPath) {
-        const openAPISchema = yaml.load(fs.readFileSync(schemaPath, defaultEnconding));
+        const defaultEncoding = 'utf8';
+        const openAPISchema = yaml.load(fs.readFileSync(schemaPath, defaultEncoding));
```

Comment:
Just hardcode it since it only used once:

```suggestion
        const openAPISchema = yaml.load(fs.readFileSync(schemaPath, 'utf8'));
```